src/common/utils.ts

```diff
@@ -2,78 +2,22 @@ import turfIntersect from '@turf/intersect';
 import turfBBoxPolygon from '@turf/bbox-polygon';
 import turfLineStringToPolygon from '@turf/line-to-polygon';
 import { Geometry, Feature, Polygon, MultiPolygon, LineString, feature as turfFeature, BBox } from '@turf/helpers';
-import { Tile } from '../tiles/tile';
-import { COORDINATE_SYSTEM, TILE_AXIS_SIZE } from './constants';
-import { Coordinate } from './interfaces/interfaces';
-
-export function getTileResolution(zoomLevel: number): number {
-  return COORDINATE_SYSTEM.maxLon / (1 << zoomLevel);
-}
-
-export function getPixelResolution(zoomLevel: number): number {
-  const tileRes = getTileResolution(zoomLevel);
-  return tileRes / TILE_AXIS_SIZE;
-}
-
-export function degreesPerTile(zoomLevel: number): number {
-  const latRange = 180;
-  return latRange / (1 << zoomLevel);
-}
+import { degreesPerPixel, snapBBoxToTileGrid } from '@map-colonies/mc-utils';
```

Comment:
delete unused values from ./constants (COORDINATE_SYSTEM, TILE_AXIS_SIZE) check if other values are still needed ({ Coordinate } from './interfaces/interfaces, { Tile } from '../tiles/tile) etc.

Reply:
is tilesCountPerZoom exported from mc-utils?